app/assets/json/translations/react/en.json

```diff
@@ -844,7 +844,7 @@
   "listings.householdMaximumIncome": "Household Maximum Income",
   "listings.housingProgram": "Housing Program",
   "listings.importantProgramRules": "Important Program Rules",
-  "listings.includesPriorityUnits": "Includes Priority Units for %{priorities}",
+  "listings.includesPriorityUnits": "Includes Priority Units for:",
```

Comment:
Thank you for digging into these strings! I would honestly recommend deleting them from the es, tl, and zh files, just to make sure that we get the translation correct.